services/create_order_list.rb

```diff
@@ -0,0 +1,20 @@
+class OrderListNotClosed < Exception; end
+
+class CreateOrderList
+  def initialize(params)
+    @params = factor_params(params)
+  end
+
+  def call
+    place = Place.where(name: @params).first
+    if OrderList.last && !OrderList.last.closed
```

Comment:
I think that this could be moved to private method in this service. And make it self explainatory so it will be more readable